src/main/java/fr/insee/genesis/domain/service/context/DataProcessingContextService.java

```diff
@@ -168,9 +168,9 @@ public DataProcessingContextModel getContext(String interrogationId) throws Gene
     }
 
     @Override
-    public DataProcessingContextModel getContextByCollectionInstrumentId(String partitionId){
+    public DataProcessingContextModel getContextByCollectionInstrumentId(String collectionInstrumentId){
         return DataProcessingContextMapper.INSTANCE.documentToModel(
-                dataProcessingContextPersistancePort.findByPartitionId(partitionId)
+                dataProcessingContextPersistancePort.findByPartitionId(collectionInstrumentId)
```

Comment:
collectionInstrumentId is not a partition. Maybe change the def of "findByPartitionId" ?